src/openrct2/ride/Track.h

```diff
@@ -79,7 +79,7 @@ struct PreviewTrack
     int16_t x;     // 0x01
     int16_t y;     // 0x03
     int16_t z;     // 0x05
-    uint8_t var_07;
+    uint8_t VerticalSize;
```

Comment:
To match the style of the codebase this would be called `ClearZ` or `ClearanceZ`

Reply:
Changed it to `ClearanceZ`.